index.html

```diff
@@ -5,45 +5,364 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+   <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Lato">
+
   <style>
     .header {
-      text-align: center;
+      text-align: left;
+      margin-top: 70px;
+     
     }
+
+    .income {
+      font-color: #F2998F;
+    }
+
+    .bar {
+          height: 20px;
+          background: #F2998F;
+        }
+      
+    .pager>li.previous>a {
+      color: #000000;
+    }
+
+    .pager>li.next>a {
+      color: #000000;
+    }
+
+   table {
+      color:  #ffffff;
+      margin-top: 50px;
+   
+    }
+
+    th {
+      font-size: 16px;
+    }
+
+    body {
+      background: #357C8D;
+      color: #ffffff;
+      font-family: "Lato";
+
+    }
+
+    h1 {
+      font-family: "Lato";
```

Comment:
`h1` and everything else inherits its font from `body`, so we don't need to set this again.

Also, it's good to have a backup in case the font doesn't download. So you might change this to `font-family: "Lato", sans-serif;`.